packages/blade/src/components/Accordion/AccordionItem.tsx

```diff
@@ -38,6 +38,20 @@ type AccordionItemProps = {
   _index?: number;
 } & TestID;
 
+/**
+ * On React Native if the `AccordionItem` has a lengthy description which renders a `Text` spanning multiple lines,
+ * it sometimes messes up the layout calculation (it thinks of multiline as a single line before flowing in the UI).
+ * And during the expanding / collapsing animation, text reflows causing words to jump around across lines.
+ *
+ * Rendering a blank `Text` at the end seems to fix all this 🤯
```

Comment:
Should we add a TODO "Check if it's also happening in RN v70+" if not we could probably remove these hacks once we upgrade RN.

Reply:
Don't think it's react native version specific issue

Reply:
Ohh okay, can confirm later, asking because I had seen layout measurement issues while working on Tooltip. It only happend on our RN version.